WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1612,6 +1612,9 @@ public void updatePostContent(boolean isAutoSave) throws IllegalEditorStateExcep
                 // A core android bug might cause an out of bounds exception, if so we'll just use the current editable
                 // See https://code.google.com/p/android/issues/detail?id=5164
                 postContent = new SpannableStringBuilder(StringUtils.notNullStr((String) mEditorFragment.getContent()));
+            } catch (RuntimeException e) {
```

Comment:
The catch for `IndexOutOfBoundsException` is now redundant, since that is a `RuntimeException`.